OnlineStore/OnlineStore/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+namespace OnlineStore
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Furniture sofa = new Furniture("Sofa", "red", 300);
```

Comment:
U can create an array, and while doing it, dynamically, create new needed objects inside it, so the whole "Assignment" code block would be unnecessary